src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -3147,6 +3162,16 @@ void CacheManager::BackgroundBackuper::DoBackup() {
   backup_notifier_.NotifyOne();
 }
 
+void CacheManager::BackgroundBackuper::WaitForBackupIsDone() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock auto_lock(backup_done_lock_);
+  if (!backup_is_in_progress_) {
```

Comment:
@AKalinich-Luxoft I would rewrite it in more simple way:
```cpp
  if (backup_is_in_progress_) {
    backup_done_.Wait(auto_lock); 
  }
```
instead of this one: 

```cpp
  if (!backup_is_in_progress_) {
    return;
  }
  backup_done_.Wait(auto_lock);
```

Can we do it?

Reply:
@AByzhynar yes, looks much cleaner. Thanks
Fixed in https://github.com/smartdevicelink/sdl_core/pull/3099/commits/4a60b806dbcad635a765411ad4588b01017baffc